numba/cuda/tests/cudadrv/test_pinned.py

```diff
@@ -21,7 +19,12 @@ def _run_copies(self, A):
         self.assertTrue(np.allclose(A, A0))
 
     def test_pinned(self):
-        A = np.arange(2 * 1024 * 1024) # 16 MB
+        machine = platform.machine()
+        if machine.startswith('arm') or machine.startswith('aarch64'):
+            count = 262144   # 2MB
+        else:
+            count = 2097152  # 16MB
```

Comment:
Assumes an 8 byte `intp`? Given a 4 byte `intp` would lead to a smaller allocation size, I think this is still fine.

Reply:
Indeed. All systems that support CUDA are 64-bit anyway.